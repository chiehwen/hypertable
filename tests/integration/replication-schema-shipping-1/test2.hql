USE "/source";
ALTER TABLE Repl ADD (d);
