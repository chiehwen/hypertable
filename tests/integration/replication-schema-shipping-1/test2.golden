
Welcome to the hypertable command interpreter.
For information about Hypertable, visit http://hypertable.com

Type 'help' for a list of commands, or 'help shell' for a
list of shell meta commands.

USE "/rep";
GET LISTING;
source	(namespace)
USE "source";
GET LISTING;
Repl
SHOW CREATE TABLE Repl;

CREATE TABLE Repl (
  c,
  d,
  ACCESS GROUP default (c, d)
)

